packages/frontend-2/nuxt.config.ts

```diff
@@ -19,12 +15,11 @@ export default defineNuxtConfig({
       }
     ]
   ],
-
   runtimeConfig: {
     public: {
-      API_ORIGIN,
-      MIXPANEL_API_HOST,
-      MIXPANEL_TOKEN_ID
+      apiOrigin: 'foo',
```

Comment:
i'm guessing the values will be changed by .env values during runtime, right? if so we should still set some kind of sensible defaults or at least something like 'UNDEFINED' so that if we ever forget to set the env vars we get more reasonable default values